compiler/include/expr.h

```diff
@@ -466,6 +466,10 @@ static inline bool needsCapture(FnSymbol* taskFn) {
          taskFn->hasFlag(FLAG_NON_BLOCKING);
 }
 
+static inline Symbol* outerVarSym(ShadowVarSymbol* sv) {
+  return sv->outerVarSE()->symbol();
+}
```

Comment:
It's not obvious to me why this should be a function and not a method on ShadowVarSymbol. It's obvious it should be viewed as a "convenience method" though.

Reply:
Good point. I will switch it to a convenience method.